_ingest-pipelines/processors/foreach.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: default
+title: Foreach
+parent: Ingest processors
+nav_order: 110
+---
+
+# Foreach processor
+
+The `foreach` processor is used to iterate over a list of values in an input document and perform some operation on each value. This can be useful for tasks like extracting information from a nested JSON structure or applying transformations to a collection of fields.
```

Comment:
Extracting information from a nested JSON structure is not a valid case, the most important use case is to process all the elements in an array in the same way, for example, lowercase or uppercase all the elements in a string array.